### DISCUSSION TITLE
Add support for podman

### DISCUSSION THREAD

**Discussion by digiserg** [axonops/axonops-workbench#498] [Ideas]
Whilst testing on RockyLinux which uses podman by default I noticed it does not work

**Comment by millerjp** [comment]
We may need to look at this @digiserg as the licencing on docker is preventing people leveraging the local cluster option.

**Comment by digiserg** [comment]
I had it running using podman by creating a simple shell script in the path (`/usr/local/bin/docker` for example)

```sh
#!/bin/bash
podman $@
```

**Comment by millerjp** [comment]
Ok - we will pull this into the next release see #501
